core/jazz_acl/swagger/swagger.json

```diff
@@ -0,0 +1,1596 @@
+              }
+            }
+          }
+        }
+      }
+    },
+    "/{service_name}/policies": {
+      "options": {
+        "description": "CORS Support",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "CORS"
+        ],
+        "x-amazon-apigateway-integration": {
+          "type": "mock",
+          "requestTemplates": {
+            "application/json": "{\"statusCode\" : 200}"
+          },
+          "responses": {
+            "default": {
+              "statusCode": "200",
+              "responseTemplates": {
+                "application/json": "$input.json('$')"
+              },
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Default response for CORS method",
+            "headers": {
+              "Access-Control-Allow-Headers": {
+                "type": "string"
+              },
+              "Access-Control-Allow-Methods": {
+                "type": "string"
+              },
+              "Access-Control-Allow-Origin": {
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/400ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/401ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {
+              "$ref": "#/definitions/403ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/404ErrorResponse"
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "$ref": "#/definitions/500ErrorResponse"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "acl"
+        ],
+        "description": "Post method to add policies to a user on service",
+        "operationId": "addPolicy",
+        "consumes": [
+          "application/json",
+          "application/xml"
+        ],
+        "parameters": [{
+          "name": "body",
+          "in": "body",
+          "description": "list of <user,serviceId,permission>",
+          "required": true,
+          "schema": {
```

Comment:
As discussed, this should change  a little bit. Lot of complexity can be reduced if you handle updates to policies by expecting payload in the format below - 

```json
{
	"serviceId": "foobar",
	"policies": [
	  {
	    "user": "john",
	    "permission": "admin",
	    "category": "manage"
	  },
	  {
	    "user": "user",
	    "permission": "write",
	    "category": "code"
	  },
	  {
	    "user": "doe",
	    "permission": "write",
	    "category": "manage"
	  }
	]
}
```

Reply:
updated